index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/assets/favicon_io/favicon-32x32.png">
+    <link rel="stylesheet" href="/css/style.css" />
+    <title>Omnifood</title>
+  </head>
+  <body>
+    <div class="header">
+      <img src="/assets/omnifood-logo.png" alt="img" class="img">
+      <div class="header-right">
+        <a class="active" href="#home">Home</a>
+        <a href="#menu">Menu</a>
+        <a href="#del">delivery</a>
+        <a href="#review">Reviews</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+        <button type="button" class="button">Sign in</button>
```

Comment:
remove the button here. The last navigation menu is 'Try for free', so design that navigation link like that will look like a button. No need for the 'Sign In' button for now.